resources/views/checkout/partials/sections/payment.blade.php

```diff
@@ -1,27 +1,36 @@
 <x-rapidez-ct::card.inactive>
-    <form id="payment" class="flex flex-col gap-2" v-on:submit.prevent="save(['payment_method'], 4)">
+    <form
+        class="flex flex-col gap-2"
+        id="payment"
+        v-on:submit.prevent="save(['payment_method'], 4)"
+    >
         <div v-for="(method, index) in checkout.payment_methods">
             <x-rapidez-ct::input.radio
+                class="min-h-[40px]"
```

Comment:
This seems like an oddly specific value, where does this number come from? 
Do we need this min height and couldn't it be solved using non-jit classes? (e.g. `min-h-10`)